### PR TITLE
Export unconnected widget classes

### PR DESCRIPTION
**Pull request by tiberiuichim**
Allows extending and overriding the connected props. More flexibility is good. Need to do this for all connected components, right now I've only edited the widgets.

### CODE REVIEW

**Comment by tiberiuichim** [comment]
Fixes #1076

File: api/plone-5.2.x.cfg [hunk 1]
```diff
@@ -8,3 +8,4 @@ versions=versions
 
 [versions]
 plone.restapi =
+lxml = 4.4.2
```
Comment (sneridagh) [inline api/plone-5.2.x.cfg]: Please do not mix things in PRs. If this is needed because some other reason, please create a PR for it.

**Comment by sneridagh** [comment]
@tiberiuichim Jus saw the issue related, and you might have a valid point in that example. However, I'm still undecided on this one, since it adds complexity to the code that a newbie might not get right. Doing this everywhere is also a lot IMHO.

/cc @robgietema can you take a look?

**Comment by tiberiuichim** [comment]
For what is worth, for a newbie (IMHO), groking redux and ``connect`` might be bigger hurdles then the ES module system.

Would it help if I write some documentation for Volto that tackle these issues? Not that I'm an expertise, but contributing to Volto is a good cause.

**Comment by tiberiuichim** [comment]
Another personal impression: I prefer component inheritance or composition rather then override (using customizations, jbot behavior), Volto is still a young system with a lot of development, there are still things that are fixed in core components. Customizing those creates a high maintenance burden which we've already paid many times in the last months. I think it might even be easier to maintain a fork of Volto that can be mostly automatically merged, rather then keep a customized version of a component and keep it up to date with latest Volto master changes.

**Comment by tiberiuichim** [comment]
We should export these widgets as UnconnectedSomething, just like: https://github.com/plotly/react-chart-editor/blob/87e4c5dde5f32d32ca40460fdcd88527568965d4/src/components/fields/NumericOrDate.js

**Comment by tiberiuichim** [comment]
The decision was to export named objects when declaring them (such as `export class SelectWidget {}`) and to export default connected objects.

**Comment by tiberiuichim** [comment]
I think this PR is outdated. We already have the widgets in storybook, so the problem must have been fixed by now.

**Pull request closed without merging**